R/utils_assert.R

```diff
@@ -192,10 +192,13 @@ assert_nzchar <- function(x, msg = NULL) {
   }
 }
 
-assert_package <- function(package, msg = NULL) {
-  if (!requireNamespace(package, quietly = TRUE)) {
-    throw_validate(msg %||% paste("package", package, "not installed"))
-  }
+assert_package <- function(package) {
+  tryCatch(
+    check_installed(package),
```

Comment:
Would you namespace this call with `rlang::`? I know `check_installed()` is already in `@importFrom`, but I do both for the sake of defensive programming.

Reply:
Interesting, haven't seen that pattern before!